cypress/e2e/undo-redo/block-configs-should-update-flow-state/common-block-configs.cy.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * In general, e2e should focus on what are VISIBLE from users perspective.
+ * But, as we have a bunch of configs to test, we can assume that:
+ * - if a block config UI updates the flow state, it should work with the UNDO/REDO feature
```

Comment:
this is the main reason why we test the flow state instead of doing a long test like this "make a change on a config UI > assert something > undo > assert something > redo > assert something "

Reply:
I will improve the comment by providing some clarification about "Vertical e2e test" here